src/clean_air/visualise/dataset_renderer.py

```diff
@@ -86,37 +86,37 @@ def render(self):
         elif self.x_coord is None and self.y_coord is None and \
                 self.t_coord is not None:
             self.img_type = 'timeseries'
-            fig = render_plot.Plot(self.plot_list).render_timeseries()
+            fig, ax = render_plot.Plot(self.plot_list).render_timeseries()
```

Comment:
I don't know what `ax` means here, could you make the variable name more descriptive please?

Reply:
Added to allow plot styling through the notebooks, as in the `matplotlib` usage `fig, ax = plt.subplots()
`. I'll change `ax` to `axes` 👍